app/models/project.rb

```diff
@@ -6,6 +6,7 @@ class Project < ActiveRecord::Base
 
   has_many :first_choice_application_drafts,  class_name: 'ApplicationDraft', foreign_key: 'project1_id'
   has_many :second_choice_application_drafts, class_name: 'ApplicationDraft', foreign_key: 'project2_id'
+  has_many :accepted_teams, class_name: 'Team' # working on the project; after application procedure
```

Comment:
I agree, `accepted_teams` doesn't sound right. I'm not too happy with `performing_teams` (sounds a bit like a dance class :wink: ). What about `participating_teams` ?